src/language/languages.json

```diff
@@ -61,19 +61,26 @@
         "fileExtensions": ["md", "markdown", "mdown", "mkdn"],
         "blockComment": ["<!--", "-->"]
     },
-  
+
     "image": {
         "name": "Image",
         "fileExtensions": ["gif", "png", "jpe", "jpeg", "jpg", "ico", "bmp", "webp"],
         "isBinary": true
     },
-    
-    "audio": {
-        "name": "Audio",
-        "fileExtensions": ["mp3", "wav", "aif", "aiff", "ogg"],
+
+    "video": {
```

Comment:
Yeah, let's not do it in `LanguageManager`, since this isn't a language as such, but a file type.